src/styles/elements/SectionHeader/SectionHeader.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+import { colors, spacing } from '../../base';
+
+const SectionHeader = styled.h2`
+  margin-top: ${spacing.xxxxlSize};
+  padding: ${spacing.xsSize} ${spacing.xxxlSize};
+  border: ${spacing.xxsSize} solid ${colors.delta};
+  border-radius: 100px;
+
+  @media (min-width: ${spacing.desktopBreakpoint}) {
+    padding: ${spacing.sSize} ${spacing.xxxlSize};
+  }
+`;
+// any ideas how to do that border-radius effect with "%"?
```

Comment:
but what is the problem with using `%` ?

Reply:
I mean that above I used border-radius of '100px' to create border that creates a box which top and bottomm sides are straight lines and lef and right sides are a half of a circle like that:
![image](https://user-images.githubusercontent.com/37923599/76680915-6850c600-65ed-11ea-9f7f-7fc22ca8e31f.png)
I was just wondering if you know a way to use something other than just a very big number of 'px' to get sucha result?

Reply:
it's ok as is, I think that you can set a hardcoded height. It's a button :)